### PR TITLE
feat: Badge Component added with documentation

### PR DESCRIPTION
**Pull request by kotalwar-krishna**
### Badge
1. Badge component is created.
2. badge related Css you can find in badge.css file.
3. badge with avatar is created for avatar i use avatar component previeusly created.
4. button i created using anchor tag but i will change it when i will create button component.

Please Give your feedback on this PR. Please review it.

### **Screenshots** 
Badge on Avatar:
badge_on_avatar
Badge on Icon:
badge_on_icon
Badge on Button:
Badge_on_button

### CODE REVIEW

File: components/badge.css [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    /* Busy */
+}
+
+.badge-yellow{
+    color: orange;
+    /* Away */
+}
+
+.badge-grey{
+    color: grey;
```
Comment (tusharkandpal) [inline components/badge.css:68]: can use variables for different colors

File: documentation/doc.html [hunk 2]
```diff
@@ -94,10 +95,71 @@ <h3>Avtar with Text Initials</h3>
                         <div>
                             <iframe src="https://www.thiscodeworks.com/embed/61f946861e1df40015379429" style="width: 100%; height: 86px;" frameborder="0"></iframe>
                         </div>
-                <hr />
+                <!-- <hr /> -->
```
Comment (tusharkandpal) [inline documentation/doc.html]: remove this commented code

**Review by tusharkandpal — CHANGES_REQUESTED** [review]
code is very clean & class names are well written.

File: components/badge.css (hunk 1 above)
Comment (kotalwar-krishna) [reply components/badge.css:68]: this color not reused so i think i will right in this way. If i will use it in repeated manner i will use Varibles afterwords

File: documentation/doc.html (hunk 2 above)
Comment (kotalwar-krishna) [reply documentation/doc.html]: Yes I removed

File: components/badge.css (hunk 1 above)
Comment (tusharkandpal) [reply components/badge.css:68]: alright fair enough!

**Review by tusharkandpal — APPROVED** [review]
good to go!

**Review by fahd23 — COMMENTED** [review]
Nice work, naming looks good

**Pull request merged**